apps/order/schema.py

```diff
@@ -19,6 +20,12 @@ def get_cart_items_qs(info):
 
 
 def get_orders_qs(info):
+    if info.context.user.user_type == User.UserType.PUBLISHER.value:
+        return Order.objects.filter(book_order__publisher=info.context.user.publisher).exclude(
```

Comment:
Is this required?

Reply:
@thenav56  Here we have 3 cases

1. Publisher can list order belongs to that publisher
2. Admin user can list all orders (all publishers + individual users + ...)
3. Other users can list their  orders only (ie created by them only)

Reply:
I meant the exclude.

Reply:
I don't suppose publisher will order books form own store that's why I excluded that, but now I removed exclude